### ISSUE TITLE
Linux version

### ISSUE THREAD

**Issue by bilelmoussaoui** [livestyle/app#2]
It would be pretty cool to have a package version for linux (deb & rpm) 
Thanks!

**Comment by hristo-kanchev** [comment]
I agree with @bil-elmoussaoui . We need a Linux version for this. :)
